packages/server/src/db.ts

```diff
@@ -53,11 +55,27 @@ export function makeKnexConfig(dbConfig: DatabaseConfig): KnexDatabaseConfig {
 	if (dbConfig.client === 'sqlite3') {
 		connection.filename = dbConfig.name;
 	} else {
-		connection.database = dbConfig.name;
-		connection.host = dbConfig.host;
-		connection.port = dbConfig.port;
-		connection.user = dbConfig.user;
-		connection.password = dbConfig.password;
+		if(dbConfig.connectionString){
+			connection.connectionString = dbConfig.connectionString;
+		} else {
+			connection.database = dbConfig.name;
+			connection.host = dbConfig.host;
+			connection.port = dbConfig.port;
+			connection.user = dbConfig.user;
+			connection.password = dbConfig.password;
+		}
+
+		connection.ssl = {};
+		if (dbConfig.sslCaFilePath)
+			connection.ssl.ca = fs.readFileSync(dbConfig.sslCaFilePath);
+
+		if (dbConfig.sslCertFilePath && dbConfig.sslCertKeyFilePath) {
+			connection.ssl.cert = fs.readFileSync(dbConfig.sslCertFilePath);
+			connection.ssl.key = fs.readFileSync(dbConfig.sslCertKeyFilePath);
+		}
+
+		if (dbConfig.rejectUnauthorized != null)
+			connection.ssl.rejectUnauthorized = dbConfig.rejectUnauthorized;
```

Comment:
Also for consistency, please rename rejectUnauthorized to sslRejectUnauthorized

Reply:
I am not sure I can? I will test but this is the name that node-postgres is expecting? or are you meaning `dbConfig.rejectUnauthorized`? I will update that part.

Reply:
Yes `dbConfig.sslRejectUnauthorized`, since the other config properties are also called `sslSomething`